### ISSUE TITLE
get_all_field_names deprecated in newest Django update

### ISSUE THREAD

**Issue by dylanfetch** [grantmcconnaughey/django-chartit2#7]
chartit\validation.py:30: RemovedInDjango110Warning: 'get_all_field_names is an unofficial API that has been deprecated. You may be able to replace it with 'get_fields()'
  model_fields = model._meta.get_all_field_names()

I got this error while displaying a 2 axis line graph.

**Comment by grantmcconnaughey** [comment]
Does it raise an exception or just print out a warning? This library is not compatible with Django 1.10 yet since it isn't released yet.

**Comment by dylanfetch** [comment]
It's just the warning I think. That message is displayed on the log and the
border and titles of the graph are displayed, but the plot is not. Sorry if
I didn't provide enough information, I've never opened an issue on a
project before. I was unaware that I moved to an unreleased version of
Django. I updated recently while trying to resolve an issue I was having
migrating. I'll try to move back to the previous version.

Edit: I just checked and I'm using Django 1.9.7

Thank you,
Dylan Fetch
